src/background/browser-adapters/injector-adapter.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export type InjectorAdapter = {
+    injectJs(tabId, file: string, callback: Function): void;
```

Comment:
I'm worried that this may be a leaky abstraction; it essentially cannot represent a unit of functionality that can be used in isolation because the opaque tabId must come from a different, coupled implementation of a different interface. I think perhaps it would be more appropriate for these to either be methods on a "tab" interface or to be part of the same interface that the tabId comes from.